scripts/open-styleguide-pr.js

```diff
@@ -11,6 +11,7 @@ const polarisBotName = 'Shopify Polaris Bot';
 const polarisBotEmail = 'shopify-polaris-bot@users.noreply.github.com';
 const polarisBotToken = require('../secrets.json').github['shopify-polaris'];
 
+const YARN_VERSION = '10.13.0';
```

Comment:
Can we pull the version number from the yml file?

Reply:
https://www.npmjs.com/package/js-yaml or something like this?

Reply:
Oh wait... That's the node version, not Yarn. Silly me. I'll revert that last change.

Reply:
Done in de7dbfe. Can you have a look?